prefab/dependency-creation/README.md

```diff
@@ -0,0 +1,74 @@
+Creating Prefab Dependency AAR
+===============================
+
+This sample shows how to export native libraries to an AAR
+using the [Prefab] format. Generally speaking, to export native libraries in an AAR 
+with Android Gradle Plugin 4.1+, make the following changes to your Android Library project's [build.gradle]:
+
+* Enable the `prefabPublishing` build feature
+```
+    android.buildFeatures.prefabPublising true
+```
+
+* Declare the libraries and headers you wish to export in the `android.prefab`
+  block
+```
+    android.prefab.mylibrary {
+        headers "src/main/cpp/include"
+    }
+```
+* (Optional)If the exported native modules are NOT used by local Kotlin/Java modules in this AAR,
+  you could temporarily avoid [the unnecessary duplication inside AAR]
```

Comment:
Use **strong**, not YELLING

Missing space after `)`

What does "local" mean here? I think you want "required by the Kotlin/Java modules of the AAR"? Modules? Libraries? APIs? Not sure how to describe them.

Remove "temporarily", add link to the bug.

Reply:
updated: yeah, local kotlin/java modules means kotlin/java code inside this same AAR.